src/eventmodel.h

```diff
@@ -103,7 +103,8 @@ class LIBCOMMHISTORY_EXPORT EventModel: public QAbstractItemModel
         FromVCardLabel,
         ReadStatus,
         SubscriberIdentity,
-        NumberOfColumns
+        NumberOfColumns,
+        ContactName
```

Comment:
I think the idea here is that the NumberOfColumns is the last one so it indicates, well, what it says.

Strange setup though with separate roles and columns.

Reply:
Indeed, it should be the last one in the list.

Reply:
I agree, I did not notice that this is really not a role

Reply:
@pvuorela  made changes